src/caches/adams_bashforth_moulton_caches.jl

```diff
@@ -291,6 +291,7 @@ mutable struct VCAB3ConstantCache{rateType,TabType,tArrayType,rArrayType,cArrayT
   k::Int
```

Comment:
The field `k` seems superfluous when you have `step`.

Reply:
max value of `k` is 3, but `step` can go above

Reply:
so `k = min(3,step)`?

Reply:
yeah

Reply:
yeah also you are right we can write it as `k = min(3,step)`

Reply:
Maybe just change https://github.com/sipah00/OrdinaryDiffEq.jl/blob/d3e962903da86b3df80381237cebb5d0dd45afb1/src/perform_step/adams_bashforth_moulton_perform_step.jl#L582 to `cache.k = min(step, order)`?